From a463b3dd254ea3e64a898ab5facfce103b481742 Mon Sep 17 00:00:00 2001
Message-ID: <a463b3dd254ea3e64a898ab5facfce103b481742.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 23:23:56 +0000
Subject: [PATCH 15/38] gnu: go-github-com-quic-go-qtls-go1-20: Move to (gnu
 packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-quic-go-qtls-go1-20): Move from
here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: I28127466419cd798c926b62d160c6e512e5171bd
---
 gnu/packages/golang-crypto.scm | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm        | 27 ---------------------------
 2 files changed, 28 insertions(+), 27 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 22593aff19..eac4fc8139 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -3,6 +3,7 @@
 ;;; Copyright © 2019, 2020 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2022 Nicolas Graves <ngraves@ngraves.fr>
+;;; Copyright © 2023 Artyom V. Poptsov <poptsov.artyom@gmail.com>
 ;;; Copyright © 2023 Clément Lassieur <clement@lassieur.org>
 ;;;
 ;;; This file is part of GNU Guix.
@@ -255,6 +256,33 @@ (define-public go-github-com-multiformats-go-multihash
       (description "Multihash implementation in Go.")
       (license license:expat))))
 
+(define-public go-github-com-quic-go-qtls-go1-20
+  (package
+    (name "go-github-com-quic-go-qtls-go1-20")
+    (version "0.3.4")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/quic-go/qtls-go1-20")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0fl3yv1w8cygag3lav45vvzb4k9i72p92x13wcq0xn13wxirzirn"))))
+    (build-system go-build-system)
+    (arguments
+     (list
+      #:import-path "github.com/quic-go/qtls-go1-20"
+      #:go go-1.20))
+    (propagated-inputs
+     (list go-golang-org-x-crypto
+           go-golang-org-x-sys))
+    (home-page "https://github.com/quic-go/qtls-go1-20")
+    (synopsis "TLS 1.3 for QUIC")
+    (description "Go standard library TLS 1.3 implementation, modified for
+QUIC.  For Go 1.20.")
+    (license license:expat)))
+
 (define-public go-github-com-refraction-networking-utls
   (package
     (name "go-github-com-refraction-networking-utls")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 784bfe40d7..c1727c0e84 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -7088,33 +7088,6 @@ (define-public go-github-com-cheekybits-genny
     (home-page "https://github.com/cheekybits/genny/")
     (license license:expat)))
 
-(define-public go-github-com-quic-go-qtls-go1-20
-  (package
-    (name "go-github-com-quic-go-qtls-go1-20")
-    (version "0.3.4")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/quic-go/qtls-go1-20")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "0fl3yv1w8cygag3lav45vvzb4k9i72p92x13wcq0xn13wxirzirn"))))
-    (build-system go-build-system)
-    (arguments
-     (list
-      #:import-path "github.com/quic-go/qtls-go1-20"
-      #:go go-1.20))
-    (propagated-inputs (list go-golang-org-x-crypto
-                             go-golang-org-x-sys))
-    (synopsis "TLS 1.3 for QUIC")
-    (description
-     "Go standard library TLS 1.3 implementation, modified for QUIC.  For
-Go 1.20.")
-    (home-page "https://github.com/quic-go/qtls-go1-20")
-    (license license:expat)))
-
 (define-public go-github-com-quic-go-qpack
   (package
     (name "go-github-com-quic-go-qpack")
-- 
2.41.0

